main.go

```diff
@@ -188,6 +192,9 @@ The commands work as following:
    >                                                                  to stop usbmuxd and load to start it again should the proxy mess up things.
    >                                                                  The --binary flag will dump everything in raw binary without any decoding.
    ios erase [--force] [options]                                      Erase the device. It will prompt you to input y+Enter unless --force is specified.
+   ios file ls [--bundle-id=<bundleID> | --group-id=<groupID>] [--path=<path>] [--domain=<domain>] [options]  List files using RemoteXPC (iOS 17+). Requires tunnel. Use --domain to specify: app (default), group, temp, or crashlogs.
+   ios file pull [--bundle-id=<bundleID> | --group-id=<groupID>] --remote=<remotePath> --local=<localPath> [--domain=<domain>] [options] Download file using RemoteXPC (iOS 17+). Requires tunnel.
+   ios file push [--bundle-id=<bundleID> | --group-id=<groupID>] --local=<localPath> --remote=<remotePath> [--domain=<domain>] [options] Upload file using RemoteXPC (iOS 17+). Requires tunnel. Preserves source file permissions.
```

Comment:
I see we call this "--app=bundleId" in fsync.

Reply:
i removed --domain and now support `--app` `--app-group` `--tmp` and `--crash` should be much easier to use